src/styles/utils/FontUtils/fontWeight/types.ts

```diff
@@ -0,0 +1,5 @@
+import type {TextStyle} from 'react-native';
+
+type FontWeightStyles = Record<'normal' | 'medium' | 'bold', NonNullable<TextStyle['fontWeight']>>;
```

Comment:
Can this be in the same file as `FontUtils/fontWeight/index.ts`?

Furthermore, can `FontUtils/fontWeight/index.ts` be moved inline in `FontUtils/index.ts` since it's just a simple constant now?

Reply:
Unfortunately this will cause a require cycle:

```
Require cycles are allowed, but can result in uninitialized values. Consider refactoring to remove the need for a cycle.
 WARN  Require cycle: src/styles/utils/FontUtils/index.ts -> src/styles/utils/FontUtils/fontFamily/index.native.ts -> src/styles/utils/FontUtils/fontFamily/singleFontFamily/index.ts -> src/styles/utils/FontUtils/index.ts
```

Thus causing a crash in the native apps.

Reply:
But I can move the types to `FontUtils/fontWeight/index.ts`